Tests/SwiftGenKitTests/ColorsTextFileTests.swift

```diff
@@ -6,47 +6,28 @@
 
 import XCTest
 import PathKit
-import SwiftGenKit
+@testable import SwiftGenKit
 
 class ColorsTextFileTests: XCTestCase {
-  func testEmpty() {
-    let parser = ColorsTextFileParser()
-
-    let result = parser.stencilContext()
-    XCTDiffContexts(result, expected: "empty.plist", sub: .colors)
-  }
-
-  func testListWithDefaults() throws {
-    let parser = ColorsTextFileParser()
-
-    try parser.addColor(named: "Text&Body Color", value: "0x999999")
-    try parser.addColor(named: "ArticleTitle", value: "#996600")
-    try parser.addColor(named: "ArticleBackground", value: "#ffcc0099")
-
-    let result = parser.stencilContext()
-    XCTDiffContexts(result, expected: "entries.plist", sub: .colors)
-  }
-
   func testFileWithDefaults() throws {
-    let parser = ColorsTextFileParser()
-    try parser.parseFile(at: Fixtures.path(for: "colors.txt", sub: .colors))
+    let parser = ColorsFileParser()
+    parser.colors = try ColorsTextFileParser().parseFile(at: Fixtures.path(for: "colors.txt", sub: .colors))
 
     let result = parser.stencilContext()
```

Comment:
That... might make tests much more complicated, we'd have to maintain the list of colours in code whereas right now we just diff with a plist file.

For the parsing part, I'd agree with you, but how do we ensure that we've fully parsed a file? Without having a list of colours in code, and we wouldn't be able to use the context plist files as they're for the `stencilContext` part. Just test if one or 2 colours are present, and a count of the number of colours?

For the stencil context part, I'd suggest using `ColorsFileParser` as we'd normally use it:
```swift
let parser = ColorsFileParser()
parser.parseFile(at: ...)
let result = parser.stencilContext()
```